### PR TITLE
Updated to Go 1.20

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
In draft until support for Go 1.20 is added to our Actions runners

### CODE REVIEW

**Comment by berndverst** [comment]
Let's proceed

**Comment by berndverst** [comment]
Please recreate this PR - it'll be easier as discussed offline. :)

**Comment by ItalyPaleAle** [comment]
Good luck reviewing this PR 🤣 
image

**Comment by ItalyPaleAle** [comment]
Ok fixed the merge conflicts

**Review by berndverst — DISMISSED** [review]

File: .github/workflows/certification.yml [hunk 1]
```diff
@@ -205,9 +201,10 @@ jobs:
         aws-region: "${{ env.AWS_REGION }}"
 
     - name: Set up Go
+      id: setup-go
       uses: actions/setup-go@v3
       with:
-        go-version: '^${{ env.GO_VERSION }}'
+        go-version-file: 'go.mod'
```
Comment (berndverst) [inline .github/workflows/certification.yml:207]: nice

**Comment by ItalyPaleAle** [comment]
@berndverst the "/ok-to-test" is failing because it's picking up the pipeline from the master branch where the Go version was hardcoded.

**Review by berndverst — APPROVED** [review]

**Pull request merged**